shlr/windbg/windbg.c

```diff
@@ -839,6 +853,7 @@ int windbg_sync(WindCtx *ctx) {
 		ret = 0;
 		goto end;
 	}
+*/
```

Comment:
Don't leave it simply commented out if you want this merged. Have you figured out why this was done before? Pipe debug is still broken so this is hard for me to verify.